zap/src/output/luau/client.rs

```diff
@@ -210,12 +215,16 @@ impl<'src> ClientOutput<'src> {
 
 		self.indent();
 
-		let values = self.get_values(&ev.data);
+		let values = self.get_values(ev.data.len());
```

Comment:
I know we don't care about the value names in the generated code, but if we're going to the bother of writing a function I'm curious to know why you didn't just use them here?

Reply:
I didn't do it just in case the user enters a name that conflicts with other variables in the output.